assets/lokomotive-kubernetes/bare-metal/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -95,6 +96,7 @@ systemd:
           --cluster_dns=${cluster_dns_service_ip} \
           --cluster_domain=${cluster_domain_suffix} \
           --cni-conf-dir=/etc/kubernetes/cni/net.d \
+          --config=/etc/kubernetes/kubelet.config \
```

Comment:
So this is the root cause of the self-hosted kubelet not working on bare metal?

Reply:
The root cause is that we don't generate this file using systemd service, so self-hosted kubelet which use this flag cannot find the configuration file and crashes.